autode/opt/optimisers/crfo.py

```diff
@@ -99,7 +99,7 @@ def _g_norm(self) -> GradientRMS:
 
     def _initialise_run(self) -> None:
         """Initialise the optimisation"""
-        logger.info("Initialising constrained optimisation")
+        # logger.info("Initialising constrained optimisation")
```

Comment:
@t-young31 Well I subclassed CRFOptimiser to get the HybridTRMOptimiser (because I need to override self.build_internal_coordinates()), but the new optimiser is not constrained, so I thought the message would be confusing. (Using DICWithConstraints without constraints does not seem to cause any issues but it seems bad programming to use that when DIC is already present). What do you suggest?

Reply:
ah I see – thanks. what about
```suggestion
        logger.info("Initialising optimisation")
```
instead?